go.mod

```diff
@@ -288,4 +296,5 @@ replace (
 	github.com/cometbft/cometbft => github.com/celestiaorg/celestia-core v0.38.11-0.20250321105625-57e36546e8a3 // marko/core_changes_v3
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v0.50.0-beta.0.0.20250320145042-6d8b2a52fbb2 // release/v0.50.x-celestia
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
+	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.45.0-tm-v0.34.35
```

Comment:
weird, how did tendermint/tendermint get included? is this something to do with embedded v3?

Reply:
yes, nova needs to create an abci client for tm v0.34.

Reply:
ah ofc, that makes sense. Kind of lucky with the change to cometbft then, if it was still tendermint/tendermint we would not be able to import, right...

Reply:
Yeah, it would have to be semver. We could have otherwise copied the whole abci protos instead too.

Given this is for celestia only, this wasn't needed. To make it more general it would be the way to go.